src/pages/generalSettings/GeneralSettings.jsx

```diff
@@ -297,35 +292,10 @@ export default function GeneralSettings() {
           )}
           <Button
             onClick={() => {
-              /* Prepare custom fields objects to send to backend */
-              Object.values(customFields).forEach(customFieldKey => {
-                const fields = currentValues[customFieldKey];
-                if (!fields) {
-                  currentValues[customFieldKey] = {
-                    definitions: [],
-                  };
-                } else {
-                  const newFields = get(
-                    fields,
-                    'definitions',
-                    [],
-                  ).map(field => {
-                    const choices = get(field, ['schema', 'choices']);
-                    if (!choices) return field;
-                    return {
-                      ...field,
-                      options: choices.map(choice => choice.label),
-                    };
-                  });
-
-                  currentValues[customFieldKey] = {
-                    definitions: newFields,
-                  };
-                }
-              });
               putSiteSettings({ data: currentValues });
```

Comment:
before this, 
if logoPostData:
    currentValues['logo'] = logoPostData
Then we can get rid of the multiple posts

Reply:
assets used to need this special treatment. if they dont any more, that's great! try this, but if it works chances are the component logic can be simplified even more than just this

Reply:
This is part of this simplification. The only way assets are now 'special' is that you send a transaction id and get back a file guid, other than that, a file is now identical to any other site setting